src/App.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import Search from './components/Search/Search';
+import Posts from './components/Posts/Posts';
+import Title from './components/Title/Title';
+import ToggleMenu from './components/ToggleMenu/ToggleMenu';
+import { Post } from './types';
+
+import './App.css';
+
+const App = () => {
+  const filmPosts: Post[] = [
+    { id: '1', name: 'Pulp fiction' },
+    { id: '2', name: 'Bohemian Rhapsody' },
+    { id: '3', name: 'Kill Bill: Vol 2' },
+    { id: '4', name: 'Avengers: War of Infinity' },
+    { id: '5', name: 'Inception' },
+  ];
+
+  const [searchQuery, setSearchQuery] = useState<string>('');
```

Comment:
It's not necessary to define primitive types if you set default value for state (for strings, number, boolean)